.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+version: 2
+references:
+  base: &base
+    docker:
+      - image: circleci/ruby:2.6.3
+        environment:
+          RAILS_ENV: test
+          BUNDLE_PATH: '~/bitcrowd/vendor/bundle'
+    working_directory: ~/bitcrowd
+jobs:
+  tests:
+    <<: *base
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - bundler-cache-v1-{{ checksum "Gemfile.lock" }}
+            - bundler-cache-v1-
+      - run: gem install bundler
+      - run:
+          name: Install ruby dependencies
+          command: |
+            bundle check || bundle install --jobs 4 --retry 3
+            bundle clean
+      - run:
+          name: rubocop
+          command: |
+            bundle exec rubocop
+      - run:
+          name: Database Setup
+          command: |
+            bundle exec rails db:drop || true
```

Comment:
Why would you need to drop the database? My understanding was that Circle CI always spawns a fresh clean container without any data from previous runs anyway. 🤔

Reply:
👍 fixed